apis/druid/v1alpha1/supervisorspec_types.go

```diff
@@ -0,0 +1,55 @@
+/*
+
+ */
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
```

Comment:
please remove this and the below unnecessary comments

Reply:
This does exist for the druid spec types as well, should i remove those too?